recipes/libnettle/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "nettle" %}
+{% set version = "3.3" %}
+{% set sha256 = "46942627d5d0ca11720fec18d81fc38f7ef837ea4197c1f630e71ce0d470b11e" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - dlopen_test.patch  # [osx]
+
+build:
+  number: 0
+  skip: True  # [win]  
+
+requirements:
+  build:
+    - m4
+    - gmp 6.1.*
+    - toolchain3
+  run:
+    - gmp 6.1.*
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX nettle
```

Comment:
Would include some [file tests]( https://github.com/conda-forge/openblas-feedstock/blob/902cd3c64f2f42e0149677a74d6be3bfcbd10459/recipe/meta.yaml#L32-L34 ) as well. This is just a good final test to make sure we have everything we expect in the package.